components/UpdateForm/index.tsx

```diff
@@ -60,7 +61,7 @@ const UpdateFormContainer: FC<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  if (loading) return <Loading className="center-text" />
```

Comment:
This class is to center the text while the component is loading, it's causing any problem?

Reply:
Currently it looks so:
![imagen](https://user-images.githubusercontent.com/66505715/190652532-70b5a47e-7fee-4d3f-9569-d5dbd69026d7.png)

But should looks centered